cmd/kev/cmd/reconcile.go

```diff
@@ -17,34 +17,49 @@
 package cmd
 
 import (
+	"fmt"
 	"os"
 	"path"
 
 	"github.com/appvia/kev/pkg/kev"
 	"github.com/spf13/cobra"
 )
 
-func runReconcileCmd(_ *cobra.Command, _ []string) error {
+func runReconcileCmd(cmd *cobra.Command, _ []string) error {
 	cmdName := "Reconcile"
+	verbose, _ := cmd.Root().Flags().GetBool("verbose")
 
 	workingDir, err := os.Getwd()
 	if err != nil {
-		return displayError(cmdName, err)
+		return displayError(err)
 	}
 
-	manifest, err := kev.Reconcile(workingDir, os.Stdout)
+	setReporting(verbose)
+	displayCmdStarted(cmdName)
+	displayReconcileRules(verbose)
+
+	manifest, err := kev.Reconcile(workingDir)
 	if err != nil {
-		return displayError(cmdName, err)
+		return displayError(err)
 	}
 
-	os.Stdout.WriteString("...............................\n\n")
-
 	for _, environment := range manifest.Environments {
 		filePath := path.Join(workingDir, environment.File)
 		if err := writeTo(filePath, environment); err != nil {
-			return displayError(cmdName, err)
+			return displayError(err)
 		}
 	}
 
 	return nil
 }
+
+func displayReconcileRules(verbose bool) {
+	if verbose {
+		_, _ = fmt.Fprintf(os.Stdout, "\033[2mHOW DOES RECONCILE WORK?\n")
+		_, _ = fmt.Fprintf(os.Stdout, "\033[2m᛫ New services & volumes in a project will be added to all environments.\n")
+		_, _ = fmt.Fprintf(os.Stdout, "\033[2m᛫ Removed services & volumes from a project will be removed from all environments.\n")
+		_, _ = fmt.Fprintf(os.Stdout, "\033[2m᛫ Generally, updates to a project's settings DON'T affect environments.\n")
```

Comment:
Reword suggestion (combining 2 lines together) : `Updates to a project's services settings don't affect environment configurations, with exception to ports.` or  `Environment config settings takes precedence over source compose settings, with exception to ports`? Would be good to explain the impact of port change somehow?

Reply:
I'll see what I can do. Need to make sure our points are short and sweet.